src/app/reporting/Engine.cpp

```diff
@@ -489,10 +488,35 @@ void Engine::Run()
 
     if (allReadClean)
     {
-        InteractionModelEngine::GetInstance()->ReleaseClusterInfoList(mpGlobalDirtySet);
+        mGlobalDirtySet.ReleaseAll();
     }
 }
 
+bool Engine::MergeOverlappedAttributePath(ClusterInfo & aAttributePath)
+{
+    bool overlapped = false;
+    mGlobalDirtySet.ForEachActiveObject([&](auto * path) {
+        // If overlapped, we would skip this target path,
+        // --If targetPath is part of previous path, return true
```

Comment:
What is targetPath?

Reply:
i mean aAttributePath here, updated it per your suggestion